recipes/ImagingReso/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ImagingReso" %}
+{% set version = "1.6.2" %}
+{% set git_rev = "1.6.2" %}
+# {% set file_ext = "tar.gz" %}
+# {% set hash_type = "sha256" %}
+# {% set hash_value = "fd09a83b3a0903b5328ed357149fdcd4e901fd2d4714810d2a7de953e4694093" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  # '{{ hash_type }}': '{{ hash_value }}'
+  git_rev: {{ git_rev }}
+  git_url: https://github.com/ornlneutronimaging/ImagingReso.git
```

Comment:
Please don't build from git https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos